tests/model_fields/models.py

```diff
@@ -33,6 +33,11 @@ class Bar(models.Model):
     a = models.ForeignKey(Foo, models.CASCADE, default=get_foo, related_name='bars')
 
 
+class Baz(models.Model):
+    a = models.CharField(max_length=10)
+    foo = models.ForeignKey(Foo, models.CASCADE, limit_choices_to=models.Q(bars__b__startswith='a'))
+
+
```

Comment:
I saw other tests in `test_foreignkey.py` define their `Baz` classes within the scope of the test function. That does help readability.